File: backend/src/main/java/com/projektike/backend/model/Category.java
package com.projektike.backend.model;


import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import org.springframework.stereotype.Component;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Component
@Entity
public class Category {
    @Id
    @GeneratedValue
    private long id;
    private String name;
    private int price;
    private CategoryType category;
}
enum CategoryType {
    PREMIUM, DELUXE, BASIC
}

File: mihkel_vaher/frontend/src/components/Category.js
function Category(props){
    return (
        <div className="CategoryElement">
            <div className="categoryName">{props.name}</div>
            <div className="categoryPrice">{props.price}</div>
            <div className="categoryType">{props.category}</div>
        </div>
    )
}
export default Category;